packages/tests/unit-tests/test-cases/declarations/src/component-input-defaults.tsx

```diff
@@ -25,13 +25,22 @@ export class BaseProps {
   @OneWay() empty?: string;
   @OneWay() height?: number = 10;
   @OneWay() width?: number = isMaterial() ? 20 : 10;
+// TODO uncoment and fix 
+// https://trello.com/c/ZEt2d0fW/2862-fix-bug-with-inherited-default-props-generation  
+//  @Nested() baseNested = { text0: "3" }
```

Comment:
This bug might happen if nested is not typed
cause here https://github.com/DevExpress/devextreme-renovation/pull/762/files#diff-9b5fc34133a7bb7016614a2babba27b1ea346fa1a98dad79c0fb6f15ca23b41bR35
everything works like expected

Reply:
Postpone